doc/operations/recovery.rst

```diff
@@ -25,7 +25,6 @@ To initiate the first phase of the recovery procedure, one or several nodes shou
     --node-address node_ip:node_port
     --rpc-address <ccf-node-address>
     --public-rpc-address <ccf-node-public-address>
-    [--domain domain]
```

Comment:
We deprecated `--domain` a long time ago (0.13.0), but it still had semantic impact on `cchost`. This removes it entirely.